components/application-mgt/org.wso2.carbon.identity.application.mgt/src/main/java/org/wso2/carbon/identity/application/mgt/ApplicationManagementService.java

```diff
@@ -370,6 +371,16 @@ public abstract boolean isExistingApplicationTemplate(String templateName, Strin
     public abstract List<SpTemplate> getAllApplicationTemplateInfo(String tenantDomain)
             throws IdentityApplicationManagementException;
 
+    /**
+     * Get authenticators configured for an application.
+     *
+     * @param applicationID ID of an application.
+     * @return list of configured authenticators.
+     * @throws IdentityApplicationManagementException
+     */
+    public abstract AuthenticationStep[] getConfiguredAuthenticators(String applicationID, String tenantDomain)
```

Comment:
If the used applicationID is a unique UUID across the server do we need to have a tenant domain param?
If that param is required, update the method comment. It's missing